starter-code/src/clue.js

```diff
@@ -85,10 +85,126 @@ name: Patio
+  first_name:   "Victor",
+  last_name:    "Plum",
+  color:        "purple",
+  description:  "Billionare video game designer",
+  age:          22,
+  image:        "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg",
+  occupation:   "Designer",
+};
+var missScarlet = {
+  first_name:   "Kasandra",
+  last_name:    "Scarlet",
+  color:        "red",
+  description:  "She is an A-list movie star with a dark past",
+  age:          31,
+  image:        "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+  occupation:   "Actor",
+};
+var mrsPeacock = {
+  first_name:   "Eleanor",
+  last_name:    "Peacock",
+  color:        "blue",
+  description:  "She is from a wealthy family and uses her status and money to earn popularity",
+  age:          36,
+  image:        "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+  occupation:   "Socialité",
+};
 
-// Weapons Collection
-var weaponsArray = [];
+
+  var diningRoom = {name: "Dinning Room"};
+  var conservatory = {name: "Conservatory"};
+  var kitchen = {name: "Kitchen"};
+  var study = {name: "Study"};
+  var library = {name: "Library"};
+  var billiard = {name: "Billiard Room"};
+  var lounge = {name: "Lounge"};
+  var ballRoom = {name: "Ballroom"};
+  var hall = {name: "Hall"};
+  var spa = {name: "Spa"};
+  var livingRoom = {name: "Living Room"};
+  var observatory = {name: "Observatory"};
+  var theater = {name: "Theater"};
+  var guestHouse = {name: "Guest House"};
+  var patio = {name: "Patio"};
+
+
+
+  var rope = {name: "rope", weight: 10};
+  var knife = {name: "knife", weight: 8};
+  var candleStick = {name: "candlestick", weight: 2};
+  var dumbBell = {name: "dumbbell", weight: 30};
+  var poison = {name: "poison", weight: 2};
+  var axe = {name: "axe", weight: 15};
+  var bat = {name: "bat", weight: 13};
+  var trophy = {name: "trophy", weight: 25};
+  var pistol = {name: "pistol", weight: 20};
+
+
+
+      var charactersArray = [   
+        
+      ];
+
+      // Rooms' Collection
+      var roomsArray = [
+
+      ];
+
+      // Weapons Collection
+      var weaponsArray = [
+
+      ];
+
+
+charactersArray.push(mrGreen, drOrchid, profPlum, missScarlet, mrsPeacock);
+
+roomsArray.push(diningRoom, conservatory, kitchen, study, library, billiard, lounge, ballRoom, hall, spa, livingRoom, observatory, theater, guestHouse, patio);
+
+weaponsArray.push(rope, knife, candleStick, dumbBell, poison, axe, bat, trophy, pistol);
+
+var random = [
+  charactersArray,
+  weaponsArray,
+  roomsArray,
+]
+
+function randomSelector(random) {
+  var nbRandom = Math.floor(Math.random() * random.length);
+  return random[nbRandom]; 
+}
+
+function pickMistery (){
+  var weaponRandom = randomSelector(weaponsArray);
+  var roomRandom = randomSelector(roomsArray);
+  var characterRandom = randomSelector(charactersArray);
+  var misteryEnvelope = [characterRandom, weaponRandom, roomRandom]; 
+  return misteryEnvelope;
+  return random[O][1][2];
```

Comment:
There cannot be 2 return statements in a function, the second one is ignored so it is better to erase it !